src/test/predict_validation_test.cpp

```diff
@@ -165,6 +165,32 @@ TEST_F(PredictValidation, RequestWrongBatchSizeAuto) {
     EXPECT_EQ(status, ovms::StatusCode::BATCHSIZE_CHANGE_REQUIRED);
 }
 
+TEST_F(PredictValidation, RequestWrongBatchSizeAutoBinaryInputs) {
+    modelConfig.setBatchingParams("auto");
+    tensorflow::serving::PredictRequest binaryInputRequest = preparePredictRequest(
```

Comment:
Why do we need to prepare request this way if we declare it manually in line 175-179?